src/components/svelte-react.ts

```diff
@@ -67,6 +45,58 @@ export type ReactProps<Props, Events> = Props & {
   [P in IntrinsicProps]?: JSX.IntrinsicElements['div'][P]
 }
 
+const useEventHandlers = (props: any) => {
+  const [el, setEl] = useState<HTMLElement>()
+  const lastValue = useRef<{ [key: string]: (...args: any[]) => any }>({})
```

Comment:
Would be good to comment what we're storing in `lastValue` - last value of what? Maybe `currentEventHandlers`?